app/server/utils/ussd/token_processor.py

```diff
@@ -56,44 +75,73 @@ def transfer_token(sender: User, recipient: User, amount: float, reason_id: Opti
 
     @staticmethod
     def send_balance_sms(user: User):
-        balance = TokenProcessor.get_balance(user)
-        # TODO: send sms
+        def get_token_info(transfer_account: TransferAccount):
+            token = transfer_account.token
+            limit = TokenProcessor.get_limit(user, token)
+            return {
+                # TODO: is it name or symbol?
+                "name": token.name,
+                "balance": transfer_account.balance,
+                "limit": limit.transfer_balance_percentage,
+                "exchange_rate": TokenProcessor.get_exchange_rate(user, token)
+            }
+        # TODO: should we merge transfer accounts with same tokens..? any currencies we should filter out?
```

Comment:
I wanna say no to the first one, but then we'd need a way of distinguishing them to the user... we could probably pull the transfer account's associated organisation name...

no filtering as far as i know